src/main/res/layout/profile_avatar_item.xml

```diff
@@ -0,0 +1,57 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Some small tweaks: Can we get
- a bit more spacing over&under the avatar, and a bit less spacing under "Last seen"?
- a bit more spacing left and right of the "Apps&Media" and "Send Message" icons?
- Slightly larger font for "Last seen", addresses, and "Introduced by"? I think they are hard to read esp. for people with impaired vision. And esp. "Last seen", people will sometimes want to know, e.g. for removing inactive members from groups.

Reply:
i tweaked spacing a bit